pkg/providers/instancetype/suite_test.go

```diff
@@ -179,8 +181,9 @@ var _ = Describe("Instance Types", func() {
 			"topology.ebs.csi.aws.com/zone": "test-zone-1a",
 		}
 
+		expected := v1alpha5.WellKnownLabels.Clone().Delete(v1.LabelWindowsBuild)
```

Comment:
Any reason this label needs to be pulled out and can't be put in the label set for the test case?